docker-compose.yml

```diff
@@ -6,4 +6,16 @@ services:
       - "5432:5432"
     restart: on-failure
     env_file:
-      - ./discord_bot/discord_db/.env
\ No newline at end of file
+      - ./discord_bot/discord_db/.env
+  
+  grafana:
+    image: grafana/grafana:latest
+    ports:
+      - "3000:3000"
+    volumes:
+      - grafana_data:/var/lib/grafana
+    env_file:
+      - .env
+
+volumes:
```

Comment:
Again not sure why you are using volumes here.

Reply:
I read its recommended to use them for backup in case the container restarts

Reply:
Ok, so just change the version above and we can merge

Reply:
done